setup.py

```diff
@@ -64,6 +74,6 @@
       extras_require={'test': tests_require},
       entry_points={
           'console_scripts': [
-              'flask=raster_server.server:run',
```

Comment:
Wel even checken of dit niet op de server gebruikt wordt, weet ik niet zeker

Reply:
Server doet bin/gunicorn voor de raster server en bin/django run_gunicorn voor de webserver.